client/src/endpoint/transaction.rs

```diff
@@ -4,6 +4,7 @@ use std::str::FromStr;
 use stellar_resources::Transaction;
 use super::{Body, EndPoint, Order, Records};
 use http::{Request, Uri};
+pub use super::account::Transactions as ForAccount;
```

Comment:
NICE

Reply:
That was a @kbacha special! He gave me the idea. Turns out nice, it matches the API in the JS sdk